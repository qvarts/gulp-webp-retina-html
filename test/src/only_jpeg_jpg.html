<!doctype html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport"
        content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Only jpeg and jpg</title>
</head>
<body>
<img class="test-class" src="../img/test.jpeg" alt="test" />
<img class="test-class" src="../img/test.jpg" alt="test" />
<img class="test-class" src="../img/test.png" alt="test" />
<img class="test-class" src="../img/test.gif" alt="test" />
<img class="test-class" src="../img/test.svg" alt="test" />
</body>
</html>