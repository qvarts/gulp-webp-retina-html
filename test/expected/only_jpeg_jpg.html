<!doctype html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <meta name="viewport"
        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Only jpeg and jpg</title>
</head>
<body>
<picture>
<source srcset="/img/test.webp, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test@2x.jpeg 2x" type="image/jpeg">
<img class="test-class" src="/img/test.jpeg" alt="test" />
</picture>
<picture>
<source srcset="/img/test.webp, /img/test@2x.webp 2x" type="image/webp">
<source srcset="/img/test@2x.jpg 2x" type="image/jpeg">
<img class="test-class" src="/img/test.jpg" alt="test" />
</picture>
<img class="test-class" src="/img/test.png" alt="test" />
<img class="test-class" src="/img/test.gif" alt="test" />
<img class="test-class" src="/img/test.svg" alt="test" />
</body>
</html>